### PR TITLE
Regen runtime proto clients and update codes accordingly

### PR DESCRIPTION
**Pull request by youngbupark**
# Description

Regen runtime proto clients and update codes accordingly

* Regen runtime proto clients
* Update codes based on new proto clients
* Remove multiple imports for the same pkg 
* Rename `internal` proto directory to `internals` to prevent go from treating it as internal.
* Add more comments to proto

> Note: E2E test needs to run after this PR merged. otherwise, e2e test will fail because e2e test is referring to master branch proto clients.

## Issue reference

#1464 

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [x] Created/updated tests
* [x] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

File: dapr/proto/runtime/v1/appcallback.proto [hunk 1]
```diff
@@ -25,25 +25,22 @@ service AppCallback {
   // Lists all topics subscribed by this app.
   rpc ListTopicSubscriptions(google.protobuf.Empty) returns (ListTopicSubscriptionsResponse) {}
 
-  // Listens events from Pubsub
+  // Subscribes events from Pubsub
   rpc OnTopicEvent(TopicEventRequest) returns (google.protobuf.Empty) {}
 
   // Lists all input bindings subscribed by this app.
-  rpc ListBindingsSubscriptions(google.protobuf.Empty) returns (ListBindingsSubscriptionsResponse) {}
+  rpc ListInputBindings(google.protobuf.Empty) returns (ListInputBindingsResponse) {}
```
Comment (youngbupark) [inline dapr/proto/runtime/v1/appcallback.proto:32]: @yaron2 Please review this naming. Using `InputBindings`  explicitly seems more readable than `BindingsSubscriptions`. if you have any objections for this, I will revert it. 

cc/ @amanbha

Comment (yaron2) [reply dapr/proto/runtime/v1/appcallback.proto:32]: I'm fine with this change.

File: dapr/proto/common/v1/common.proto [hunk 2]
```diff
@@ -104,14 +104,15 @@ message StateOptions {
...
-    LAST_WRITE = 2;
+    CONCURRENCY_FIRST_WRITE = 1;
+    CONCURRENCY_LAST_WRITE = 2;
   }
 
   // Enum describing the supported consistency for state.
   enum StateConsistency {
-    EVENTUAL = 0;
-    STRONG = 1;
+    CONSISTENCY_UNSPECIFIED = 0;
```
Comment (amanbha) [inline dapr/proto/common/v1/common.proto:113]: does it need extra CONSISTENCY_ here?

File: dapr/proto/common/v1/common.proto [hunk 3]
```diff
@@ -104,14 +104,15 @@ message StateOptions {
   // Enum describing the supported concurrency for state.
   enum StateConcurrency {
     CONCURRENCY_UNSPECIFIED = 0;
-    FIRST_WRITE = 1;
-    LAST_WRITE = 2;
+    CONCURRENCY_FIRST_WRITE = 1;
```
Comment (amanbha) [inline dapr/proto/common/v1/common.proto:107]: does it need extra CONCURRENCY_ here

File: dapr/proto/common/v1/common.proto [hunk 4]
```diff
@@ -124,8 +125,8 @@ message StateRetryPolicy {
   // Enum describing the support retry pattern
   enum RetryPattern {
     RETRY_UNSPECIFIED = 0;
-    LINEAR = 1;
-    EXPONENTIAL = 2;
+    RETRY_LINEAR = 1;
```
Comment (amanbha) [inline dapr/proto/common/v1/common.proto:128]: Does in need extra RETRY_ here?

File: dapr/proto/common/v1/common.proto (hunk 3 above)
Comment (youngbupark) [reply dapr/proto/common/v1/common.proto:107]: this is the [recommended way](https://developers.google.com/protocol-buffers/docs/style#enums). In the generated client codes, we can refer to the enum value like below. 

commonv1pb.StateOptions_CONCURRENCY_LAST_WRITE

In the case that we do not add the prefix, we can refer to the enum value like below,
commonv1pb.StateOptions_LAST_WRITE

Regardless of recommendations, having CURRENCY_ prefix looks more readable to me.

How do you think about it?

cc/ @yaron2

File: dapr/proto/common/v1/common.proto (hunk 4 above)
Comment (youngbupark) [reply dapr/proto/common/v1/common.proto:128]: Please see the previous comment.

File: dapr/proto/common/v1/common.proto (hunk 3 above)
Comment (amanbha) [reply dapr/proto/common/v1/common.proto:107]: Makes sense, sounds good

**Comment by youngbupark** [comment]
@yaron2 I will merge this pr and then run e2e test. Please let me know if you have any objections.

**Pull request merged**